main/load-tx-rlp.zkasm

```diff
@@ -333,7 +334,7 @@ appendTxs:
 
 finalAppendTxs:
         B - C => D
-        D - 1                           :JMPN(endAppendTxs)
+        D - 1                           :JMPN(endAppendTxs) ; TODO check, JUMPN only chekcs first 64 bits
```

Comment:
```suggestion
        D - 1                           :JMPN(endAppendTxs) ; TODO check, JUMPN only chekcs first 32 bits
```

Reply:
I would remove the comments because the same reason as `load-tx-rlp-utils.zkasm`